user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Logging/Dashboards_and_Low_Code_Apps_Logging.md

```diff
@@ -0,0 +1,50 @@
+---
+uid: Dashboards_and_Low_Code_Apps_Logging
+---
+
+# Logging
+
+The logging for Dashboards and Low Code Apps is located in `C:\Skyline DataMiner\Logging\Web`. There are two distinct types of logging, each serving specific purposes:
```

Comment:
Since this is introduced in DataMiner 10.3.0 [CU14]/10.4.0 [CU2]/10.4.5, could you add the necessary version info to your page? Otherwise, users with earlier DataMiner versions will be confused if they can't find the logging.